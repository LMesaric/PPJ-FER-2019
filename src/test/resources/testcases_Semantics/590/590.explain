semantic: d value not int
note: 'd + dd' on line 5
