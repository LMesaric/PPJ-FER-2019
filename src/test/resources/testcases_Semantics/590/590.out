<aditivni_izraz> ::= <aditivni_izraz> PLUS(5,+) <multiplikativni_izraz>
